lib/benchee/conversion/memory.ex

```diff
@@ -46,6 +46,43 @@ defmodule Benchee.Conversion.Memory do
               }
   }
 
+  # Unit conversion functions
+
+  @type unit :: Unit.t
```

Comment:
In general, when a type like this has been defined in another module, it's usually cool to reference it directly. In this case, I think this type is unnecessary since it's essentially duplicating the name of the module on which the type is defined.

So, for example, in your typespec on line 74 you could write:

```elixir
@spec convert(value_with_unit, Unit.t()) :: value_with_unit
```

Reply:
Makes sense to me. Will change accordingly